app/javascript/modCenter/singleArticle/index.jsx

```diff
@@ -51,13 +59,16 @@ export default class SingleArticle extends Component {
       >
         <span className="article-title">
           <header>
-            <h3 className="fs-base fw-bold lh-tight">{title}</h3>
+            <h3 className="fs-base fw-bold lh-tight">
+              <a href={path}>{title}</a>
+            </h3>
           </header>
           {tags}
         </span>
         <span className="article-author fs-s lw-medium lh-tight">
           {newAuthorNotification}
           {user.name}
+          {/* <a href={user.username}>{user.name}</a> */}
```

Comment:
```suggestion

```
I think this should be removed? Or should this still be a link to the author's profile?

Reply:
So while in testing on my own local host 3000, I was able to click on the author's name which would then take me to the author's profile. Problem is, while committing the file it didn't pass the test cases in singleArticle.text.jsx, specifically the one on lines 73 to 77 where it renders the author's name. I just commented it out and put the {user.name} back on line 70. Otherwise you would replace line 70 with 71 if it works.

Reply:
I've committed a newer version removing the comment.